doc/flatpak-metadata.xml

```diff
@@ -253,6 +268,54 @@
                                 Available since 0.3.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>host-os</option></term>
+                            <listitem><para>
+                                The host operating system's libraries,
+                                executables and static data from
+                                <filename>/usr</filename>
+                                and the related directories
+                                <filename>/bin</filename>,
+                                <filename>/lib</filename>,
+                                <filename>/lib32</filename>,
+                                <filename>/lib64</filename>,
+                                <filename>/sbin</filename>.
+                                Additionally, this keyword provides access
+                                to a subset of <filename>/etc</filename> that
+                                is associated with packaged libraries and
+                                executables, even if the
+                                <option>host-etc</option> keyword
+                                was not used:
+                                <filename>/etc/ld.so.cache</filename>,
```

Comment:
I'm unsure whether we should also expose `/etc/ld.so.conf` and `/etc/ld.so.conf.d`. I don't currently need them for my work with Steam, though.

Reply:
I don't think those are needed as long as you have the compiled result (i.e. ld.so.cache).

Reply:
That has certainly been my experience in libcapsule and pressure-vessel.

You only need `ld.so.conf` if you want to regenerate the cache with `ldconfig`, which the app isn't allowed to do anyway.